src/Maker/MakeResetPassword.php

```diff
@@ -194,23 +203,94 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             ]
         );
 
-        $generator->generateClass(
-            $requestClassNameDetails->getFullName(),
-            'resetPassword/ResetPasswordRequest.tpl.php',
-            [
-                'repository_class_name' => $repositoryClassNameDetails->getFullName(),
-                'user_full_class_name' => $userClassNameDetails->getFullName(),
-            ]
+        $entityClassGenerator = new EntityClassGenerator($generator, $this->doctrineHelper);
+
+        $pathRequestEntity = $entityClassGenerator->generateEntityClass($requestClassNameDetails, false);
+
+        $generator->writeChanges();
+
+        $manipulator = new ClassSourceManipulator(
+            $this->fileManager->getFileContents($pathRequestEntity)
         );
 
-        $generator->generateClass(
-            $repositoryClassNameDetails->getFullName(),
-            'resetPassword/ResetPasswordRequestRepository.tpl.php',
-            [
-                'request_class_full_name' => $requestClassNameDetails->getFullName(),
-                'request_class_name' => $requestClassNameDetails->getShortName(),
-            ]
+        $manipulator->addUseStatementIfNecessary(ResetPasswordRequestInterface::class);
+
+        $manipulator->addUseStatementIfNecessary(ResetPasswordRequestTrait::class);
+
+        $manipulator->addInterface('ResetPasswordRequestInterface');
+
+        $manipulator->clearClassNodeStmts();
+
+        $manipulator->addTrait('ResetPasswordRequestTrait');
```

Comment:
Inside this method, IT should handle adding the use statement. So we should not need the `addUseStatementIfNecessary` above this.

The same is true for `addInterface()`: THAT method should add the `use` statement *and* add implement the interface.

Reply:
Done